### PR TITLE
fix(nuxt): nuxt v3 compatibility

### PR DESCRIPTION
**Pull request by pi0**
Update to RC.14 with latest api changes and dependency bumps (major) to prepare for v3.0.0

https://github.com/nuxt/framework/releases/tag/v3.0.0-rc.14

### CODE REVIEW

File: packages/nuxt/src/module.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {
   defineNuxtModule,
   addPlugin,
   isNuxt2,
-  addAutoImport,
+  addImports,
   createResolver,
   resolveModule,
```
Comment (pi0) [inline packages/nuxt/src/module.ts]: Note: `resolveModule` utility is deprecated from `@nuxt/kit` as is CJS-based resolution. But should be fine for pinia module.

Comment (posva) [reply packages/nuxt/src/module.ts]: Should I use `const runtimeDir = fileURLToPath(new URL('./runtime', import.meta.url))` + `resolve()`?

Comment (pi0) [reply packages/nuxt/src/module.ts]: We could replace it with [`mlly.resolve`](https://github.com/unjs/mlly#resolve) that implements ESM resolution and supports search path with `url: []` opt (similar to `import.meta.resolve`)

Simple `resolve`, does not support search path

Comment (pi0) [reply packages/nuxt/src/module.ts]: BTW, it is nothing important IMO. We don't drop this util in Nuxt 3 only deprecating by type and current code works fine.

**Comment by pi0** [comment]
Should be good to go @posva

File: packages/nuxt/src/module.ts (hunk 1 above)
Comment (posva) [reply packages/nuxt/src/module.ts]: ok!

**Comment by brunokunace** [comment]
@pi0 have a schedule to create a new release with this version?

**Comment by abdonrd** [comment]
I just tried the new [`@pinia/nuxt@0.4.4`](https://github.com/vuejs/pinia/blob/v2/packages/nuxt/CHANGELOG.md#044-2022-11-17), but it doesn't work:

https://github.com/nuxt/framework/discussions/9032#discussioncomment-4171427

**Pull request merged**